stwfsapy/predictor.py

```diff
@@ -204,7 +206,13 @@ def plural_fun(x):
         self.pipeline_ = Pipeline([
             ("Combined Features", ColumnTransformer([
                 ("Thesaurus Features", thesaurus_features, 0),
-                ("Text Features", mk_text_features(), 1)])),
+                ("Text Features", mk_text_features(), 1),
+                ('Position Features', PositionFeatures(), [0, 1, 2, 3]),
+                (
+                    'Frequency Features',
+                    PositionFeatures(),
```

Comment:
```suggestion
                    FrequencyFeatures(),
```
Just as a note: After applying that, ``stwfsapy.tests.predictor_test.test_serialization_inversion`` will fail because pickle cannot handle lambda functions (used in the defaultdict in ``stwfsapy.frequency_features.FrequencyFeatures.fit``).